engine/src/main/java/org/destinationsol/game/screens/ChooseMercenaryScreen.java

```diff
@@ -90,4 +90,17 @@ public String getHeader() {
         return "Mercenaries:";
     }
 
+    @Override
+    void onAdd(InventoryScreen inventoryScreen) {
+        inventoryScreen.setInteractText("---");
+        inventoryScreen.getInteractButton().setAction(uiElement -> {
```

Comment:
Any reason to have an empty callback? If it's incomplete as of yet, I would recommend removing this, and adding `// TODO: Set callback`